server/src/routes/candidat/places-business.spec.js

```diff
@@ -0,0 +1,57 @@
+import { connect, disconnect } from '../../mongo-connection'
+
+import {
+  createCentres,
+  removeCentres,
+  createPlaces,
+  removePlaces,
+  centres,
+} from '../../models/__tests__/'
+import { getDatesByCentre, canCancelReservation } from './places-business'
+import { getFrenchLuxon } from '../../util'
+import config from '../../config'
+
+describe('Test places business: utiles functions', () => {
+  it('Should true when entry date is 7 days and 2 hours days hours after now', () => {
```

Comment:
Il manque un verbe :
`Should return true when entry date is 7 days and 2 hours days hours after now`